File: FuncamentosProgramacao/CapituloTres/AtividadeDezesseis.cs
namespace FuncamentosProgramacao.CapituloTres;

public class AtividadeDezesseis
{
    public static void CalculaHipotenusa()
    {
        double catetoAdjacente;
        double catetoOposto;
        double hipotenusa;

        Console.Write("Informe o valor do cateto adjacente: ");
        catetoAdjacente = Double.Parse(Console.ReadLine()!);
        Console.Write("Informe o valor do cateto oposto: ");
        catetoOposto = Double.Parse(Console.ReadLine()!);

        hipotenusa = Math.Sqrt(Math.Pow(catetoAdjacente, 2) + Math.Pow(catetoOposto, 2));

        Console.WriteLine($"A hipotenusa do triângulo de cateto adjacente {catetoAdjacente}m " +
                          $"e cateto oposto {catetoOposto}m = {hipotenusa:F}m");
    }
}

File: fundamentosProgramacaoJava/src/capitulo/tres/AtividadeDezesseis.java
package capitulo.tres;

import java.util.Scanner;

public class AtividadeDezesseis {

    public static void main(String[] args) {

        float catetoAdjacente;
        float catetoOposto;
        float hipotenusa;
        Scanner leia = new Scanner(System.in);

        System.out.print("Digite o valor do catego adjacente: ");
        catetoAdjacente = leia.nextFloat();
        System.out.print("Digite o valor do catego oposto: ");
        catetoOposto = leia.nextFloat();

        hipotenusa = (float) (Math.sqrt(Math.pow(catetoOposto, 2) + Math.pow(catetoAdjacente, 2)));

        System.out.println("O valor da hipotenusa e: " + hipotenusa);

    }

}
